mapping/sustainability.yaml

```
@@ -543,8 +543,19 @@
     module: Climate finance
     indicator: Decision-making process
     disclosure format: Identify who approved the climate finance investment in the country (organization, party, role)
-    mapping: ''
-    example: ''
+    mapping: 'Project level: Add the organization to the `parties` array and add ''climateFinanceFocalPoint'' to their `.roles` array.'
```

Reference the [add an organization](https://standard.open-contracting.org/staging/infrastructure/0.9-dev/en/cost/common/#add-an-organization) common operation:

```suggestion
    mapping: 'Project level: [Add an organization](../common.md#add-an-organization) for the decision maker and add ''climateFinanceFocalPoint'' to its `.roles` array.
```